templates/taxi/manufacturer_delete_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete manufacturer model</h1>
+  <p>Are you want to delete the manufacturer models: {{ manufacturer }}</p>

```

Comment:
The sentence 'Are you want to delete the manufacturer models: {{ manufacturer }}' is grammatically incorrect. It should be 'Do you want to delete the manufacturer model: {{ manufacturer }}' to make it grammatically correct.